### PR TITLE
 Update preview tab on publisher dashboard

### PR DESCRIPTION
**Pull request by tasawernawaz**
[ECOM-7528](https://openedx.atlassian.net/browse/ECOM-7528)

* When course teams `request preview` add a status of `Preview Requested` and move the course run to the preview tab
* Change the dashboard tab from `Preview Ready` to `In Preview`
* Remove the `course number` column from this tab
* Use the organization short name, e.g. MITx, UQx, etc

**Dashboard with updated tab**

screen shot 2017-04-03 at 3 35 45 pm

### CODE REVIEW

**Comment by tasawernawaz** [comment]
@waheedahmed @awais786 Please review.

File: course_discovery/templates/publisher/dashboard/_in_preview.html [hunk 1]
```diff
@@ -31,13 +28,16 @@
...
                     <td>
-                        {{ course_run.number }}
-                    </td>
-                    <td>
-                    {% if course_run.course.organizations.first %}{{ course_run.course.organizations.first.name }}{% endif %}
+                    {% if course_run.course.organizations.first %}{{ course_run.course.organizations.first.key }}{% endif %}
                     </td>
                     <td>
-                         {{ course_run.state }}
+                        {% if not course_run.preview_url %}
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/dashboard/_in_preview.html:34]: This is not right, In review means when `preview_url` is available but not accepted.

**Review by waheedahmed — CHANGES_REQUESTED** [review]
In screenshot available on ticket it was wrong, Roderick also mentioned that in planning session last Monday.
It should be like this.
```
{% if not course_run.preview_url %}
	{% trans "Preview Requested" %}
{% elif course_run.course_run_state.preview_accepted %}
	{% trans "Approved" %}
{% else %}
	{% trans "In Review" %}
{% endif %}
```

File: course_discovery/templates/publisher/dashboard/_in_preview.html (hunk 1 above)
Comment (tasawernawaz) [reply course_discovery/templates/publisher/dashboard/_in_preview.html:34]: Done. Please review it again.

**Review by waheedahmed — APPROVED** [review]

**Pull request merged**